pkg/apis/controller/experiments/v1alpha3/constants.go

```diff
@@ -24,4 +24,7 @@ const (
 
 	// Default value of Spec.TemplatePath
 	DefaultTrialTemplatePath = "defaultTrialTemplate.yaml"
+
+	// Default value of Spec.DefaultResumePolicy
+	DefaultResumePolicy = NeverResume
```

Comment:
I think the problem with e2e is here.
Maybe it is better to specify `DefaultResumePolicy` as `LongRunning`.
It will not break current e2e implementation and it is currently working like this.

We can implement e2e test with `NeverResume` later.

Reply:
I've changed.